tests/test_cube_help.py

```diff
@@ -1,63 +1,647 @@
+                standard_name="air_potential_temperature",
+                units="K",
+                dim_coords_and_dims=[(time, 0),
+                                     (lat, 1),
+                                     (lon, 2)],
+                attributes={"source": "Iris test case"},
+            )
+            lower_bound = lower_bound + 70
+            upper_bound = upper_bound + 70
+            period = period + 70
+            cube_list.append(cube)
+        cube_list = iris.cube.CubeList(cube_list)
+        cube = cube_list.concatenate_cube()
+        return cube
+
+    def setUp(self):
+        super(TestCubeHelp, self).setUp()
+        abs_path = os.path.dirname(os.path.abspath(__file__))
+        self.tmp_dir_time = abs_path + '/' + 'tmp_dir_time/'
+        if not os.path.exists(self.tmp_dir_time):
+            os.mkdir(self.tmp_dir_time)
+        base_cube = stock.realistic_3d()
+        cube_1 = base_cube[0:2]
+        cube_2 = base_cube[2:4]
+        cube_3 = base_cube[4:]
+        new_time = cf_units.Unit('hours since 1980-01-01 00:00:00',
+                                 'gregorian')
+        cube_2.dim_coords[0].convert_units(new_time)
+        new_time = cf_units.Unit('hours since 1990-01-01 00:00:00',
+                                 'gregorian')
+        cube_3.dim_coords[0].convert_units(new_time)
+        self.temp_1_time = 'temp_1_time.nc'
+        self.temp_2_time = 'temp_2_time.nc'
+        self.temp_3_time = 'temp_3_time.nc'
+        iris.save(cube_1, self.tmp_dir_time + self.temp_1_time)
+        iris.save(cube_2, self.tmp_dir_time + self.temp_2_time)
+        iris.save(cube_3, self.tmp_dir_time + self.temp_3_time)
+
+    def test_concatenate(self):
+        glob_path = self.tmp_dir_time + '*.nc'
+        filepaths = glob(glob_path)
+        test_load = [iris.load_cube(cube) for cube in filepaths]
+        test_case_a = ch.concatenate(test_load)
+        test_load = iris.cube.CubeList(test_load)
+        test_case_b = ch.concatenate(test_load)
+        self.assertIsInstance(test_case_a, iris.cube.Cube)
+        self.assertIsInstance(test_case_b, iris.cube.Cube)
+
+    def test_load(self):
+        glob_path = self.tmp_dir_time + '*.nc'
+        filepaths = glob(glob_path)
+        directory = self.tmp_dir_time
+        test_case_a = ch.load(filepaths)
+        self.assertIsInstance(test_case_a, iris.cube.Cube)
+        self.assertEqual(test_case_a.dim_coords[0].units.origin,
+                         "hours since 1970-01-01 00:00:00")
+        self.assertEqual(test_case_a.dim_coords[0].units.calendar,
+                         "gregorian")
+        test_case_b = ch.load(directory)
+        self.assertEqual(test_case_b.dim_coords[0].units.origin,
+                         "hours since 1970-01-01 00:00:00")
+        self.assertEqual(test_case_b.dim_coords[0].units.calendar,
+                         "gregorian")
+
+    def test_add_categorical(self):
+        glob_path = self.tmp_dir_time + '*.nc'
```

Comment:
Can this test be removed now that you've got a specific test for each categorical?